### PR TITLE
Add support for PrimeKey Software Appliance

### PR DESCRIPTION
**Pull request by TheMysteriousX**
Only supports sensors. I've added support for all sensors except for some hardware specific HSM ones, as I do not have access to them to test.

For anyone using this:
* I don't know why internal database is persistently faulted
* I don't know why Utimaco HSM's are persistently faulted

Both are probably bugs in the appliance, or a check is failing that doesn't affect production usage. The sensors are reporting the data the appliance is returning accurately.

Also worth noting that this is *very* similar to the physical appliance OS definition - but due to the MIB name being different, the two are not interchangable.

(Back to draft as I haven't tested the tweaked detection rule on a real system)

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [x] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/definitions/discovery/primekey-sw.yaml [hunk 1]
```diff
@@ -0,0 +1,295 @@
+mib: PK-SOFTWARE-APPLIANCE-V2
+modules:
+    os:
+        version: PK-SOFTWARE-APPLIANCE-V2::pk-SAV2-systemVersion.0
+    sensors:
+        count:
```
Comment (murrant) [inline includes/definitions/discovery/primekey-sw.yaml:6]: This shouldn't be a count sensor, it should be storage.

Comment (TheMysteriousX) [reply includes/definitions/discovery/primekey-sw.yaml:6]: I'm not certain that it makes sense to implement this as a storage plugin, as this isn't measuring a filesystem.

I'm not entirely sure what it *is* measuring - I'm assuming there's some mechanism that limits the database to a fixed, possibly expandable file size, but it's not documented.

Happy either way though, it'd only be a few lines to convert it.

**Comment by TheMysteriousX** [comment]
Forgot to take this out of draft, worksforme.

**Review by electrocret — APPROVED** [review]
LGTM

**Pull request merged**